transaction/src/model/versioned.rs

```diff
@@ -161,7 +161,7 @@ mod tests {
         let intent_hash = prepared_intent.intent_hash();
 
         assert_eq!(
-            intent_hash.to_string(),
+            intent_hash.to_string(None),
```

Comment:
Might be interesting to also output the transaction ids eg `intent_hash.to_string(NetworkDefinition::simulator())` here :) - as this file is basically a demonstration of the various transactions.

Reply:
Sounds good, will change that.